src/imgix-core-js.js

```diff
@@ -185,21 +184,16 @@
       var srcset = '';
       var currentWidth;
       var targetWidths;
-      var widthTolerance = options["widthTolerance"] || DEFAULT_SRCSET_WIDTH_TOLERANCE;
-      var minWidth = options["minWidth"] || MIN_SRCSET_WIDTH;
-      var maxWidth = options["maxWidth"] || MAX_SRCSET_WIDTH;
-      var customWidths = options["widths"];
+      var customWidths = options.widths;
+      var srcsetOptions = validateAndDestructureOptions(options);
+      var widthTolerance = srcsetOptions[0], minWidth = srcsetOptions[1], maxWidth = srcsetOptions[2];
```

Comment:
So clunky 🤦‍♂. We should consider getting some Babel in here in the near future so we can use some more modern stuff.

Reply:
I remember this came up last year but don't recall the exact reasoning for not adding it sooner.